packages/apps-config/src/endpoints/util.ts

```diff
@@ -78,3 +83,7 @@ export function expandEndpoint (t: TFunction, { dnslink, genesisHash, homepage,
 export function expandEndpoints (t: TFunction, input: EndpointOption[], firstOnly?: boolean): LinkOption[] {
   return input.sort(sortLinks).reduce((result: LinkOption[], input) => result.concat(expandEndpoint(t, input, firstOnly)), []);
 }
+
+export function createProviderUrl (url: string, inpType = 'json-rpc'): Endpoint {
+  return { param: url, type: inpType as EndpointType };
+}
```

Comment:
```suggestion
export function createProviderUrl (param: string, type: EndpointType): Endpoint {
  return { param, type };
}
```

Where we use this we are explicit, so rather be explicit here as well. (Plus since we don't have casting, it is slightly safer)